roles/upload-site/tasks/main.yml

```diff
@@ -41,20 +41,21 @@
 # per-site basis. That will allow us to move past errors in this task, and run
 # "drush updb" a second time in the following task.
 - name: Update database schema with drush updb
-  shell: "{{drush_alias }} {{ item }}"
+  shell: "{{ drush_alias }} {{ item }}"
   with_items:
     - "-y updb"
+    - "-y rr"
```

Comment:
Won't this run rr on every site not just the failed ones?

Reply:
Right. Let’s move that to the next task.

Reply:
Before the second updb